extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -285,7 +294,7 @@ private static void installShadowFile(@NotNull final File destFile, @NotNull fin
     }
 
     /**
-     * Roll back any changes made in the {@link #installShadowFile} in a best effort manner
+     * Roll back any changes made in the {@link #installShadowFile} in the best effort manner
```

Comment:
Previous version was better. Maybe this is better than either?
```suggestion
     * Roll back any changes made in the {@link #installShadowFile} in best-effort manner
```